packages/playwright-utils/src/screenshots/utils.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Escapes the given string so its save to be used as CSS grid-area, e.g. by replacing whitespaces.
+ */
+export const escapeGridAreaName = (name: string) => {
+  return name.replace(/ /g, "-");
```

Comment:
To be a bit pedantic here 😅: 
We should probably replace all unusual characters:
```suggestion
  return name.replace(/\W/g, "-");
```